### ISSUE TITLE
OPPO R9M crash

### ISSUE THREAD

**Issue by 25juan** [react-native-elements/react-native-elements#2200]
**Explain what you did**

```jsx
import React from "react";
import { Header } from "react-native-elements"
export default (props)=><Header { ...props }/>

```

**Expected behavior**

App does not crash.

**Describe the bug**

The header component is used in my project. It will cause app to flash back at oppo R9m. I found the problem. The following changes can be made to solve the flashback problem。
in `react-native-elements/src/header/Header.js`
```
...
class Header extends Component {
   ...
  render() {
    const {
      backgroundImage,
      backgroundImageStyle,
      linearGradientProps,
      ViewComponent = linearGradientProps && global.Expo
        ? global.Expo.LinearGradient
        : (backgroundImage ? ImageBackground:View), // Modifying the previous code will cause oppo R9m to flash back. Normal operation after modification 
      theme,
      ...attributes
    } = this.props;

    return (
      <ViewComponent
        testID="headerContainer"
        {...attributes}
        style={StyleSheet.flatten([
          styles.container(theme),
          backgroundColor && { backgroundColor },
          containerStyle,
        ])}
        source={backgroundImage}
        imageStyle={backgroundImageStyle}
        {...linearGradientProps}
      >
        ...
      </ViewComponent>
    );
  }
}
 
export { Header };
export default withTheme(Header, 'Header');
```

**To Reproduce**

import React from "react";
import { Header } from "react-native-elements"
export default (props)=><Header { ...props }/>

**Screenshots**

**Your Environment (please complete the following information):**

| software              | version |
| --------------------- | ------- |
| react-native-elements |      1.2.5   |
| react-native          |  0.61.2       |

**Comment by Qiyu8** [comment]
So you mean when backgroundImage is empty(default mode),   ViewComponent should use View instead?

**Comment by 25juan** [comment]
yes。otherwise oppo  will crash

**Comment by flyingcircle** [comment]
I tried reproducing this, but I'm guessing that this only happens on certain hardware? Or perhaps the version of RN that I'm using has already fixed it? Either way, this is a pretty low risk fix, so I'll take it on several other people's word here that this fix works. Fixed by: #2199
